numpy/linalg/tests/test_linalg.py

```diff
@@ -1944,9 +1944,13 @@ def test_generalized_raise_multiloop():
 
     assert_raises(np.linalg.LinAlgError, np.linalg.inv, x)
 
+
 @pytest.mark.skipif(
     threading.active_count() > 1,
     reason="skipping test that uses fork because there are multiple threads")
+@pytest.mark.skipif(
+    NOGIL_BUILD,
+    reason="Cannot safely use fork in tests on the free-threaded build")
```

Comment:
I noticed [in this CI run](https://github.com/numpy/numpy/actions/runs/12128567744/job/33815225281#step:8:1357) that despite the skip above, CPython was still warning about an unsafe use of fork, so I think it's probably safer to just always skip this test on the free-threaded build.